pkg/KubeArmorMacro/config/crd/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+# This kustomization.yaml is not intended to be run by itself,
+# since it depends on service name and namespace that are out of this kustomize package.
+# It should be run by config/default
+resources:
+- bases/security.kubearmor.com_kubearmormacroes.yaml
```

Comment:
files are generated with "es" too, but I think it's fine. We can just focus on the contents.